bundle/terranova/tnresources/volume.go

```diff
@@ -79,6 +107,18 @@ func (r *ResourceVolume) WaitAfterUpdate(ctx context.Context) error {
 }
 
 func (r *ResourceVolume) ClassifyChanges(changes []structdiff.Change) deployplan.ActionType {
-	// TODO: Name, SchemaName changes should result in re-create
+	for _, change := range changes {
+		if change.Path.String() == ".name" {
+			return deployplan.ActionTypeUpdateWithID
```

Comment:
As an example `jobs` or `pipelines` allow changing the name but they don't implement DoUpdateWithId, how does it work with them?